odh-dashboard/base/deployment.yaml

```diff
@@ -22,12 +22,12 @@ spec:
                     - key: app
                       operator: In
                       values:
-                        - rhods-dashboard
+                        - odh-dashboard
                 topologyKey: topology.kubernetes.io/zone
-      serviceAccount: rhods-dashboard
+      serviceAccount: odh-dashboard
       containers:
-      - name: rhods-dashboard
-        image: $(odh-dashboard-image)
```

Comment:
Won't this be required to get CPaas image?

Reply:
ah damn, that's the overlay i was missing, one sec, that might be the thing i was trying to fix that i totally thought i added in the overlay!